api/services/SQS.js

```diff
@@ -85,6 +85,9 @@ const generateDefaultCredentials = build => ({
   SOURCE_REPO: sourceForBuild(build).repository,
   SOURCE_OWNER: sourceForBuild(build).owner,
   SKIP_LOGGING: config.app.app_env === 'development',
+  AUTH_BASEURL: process.env.APP_HOSTNAME,
```

Comment:
I have no idea what this does ... :)

Reply:
actually i think i know what this does now -> this is sending info to the build?

Reply:
correct